code.ts

```diff
@@ -28,14 +32,16 @@ figma.ui.onmessage = msg => {
     case 'reset':
       reset = true;
       pause = true;
+      totalTimers = 0;
+      figma.ui.resize(220, uiHeight);
       break;
 
     case 'helpon':
       figma.ui.resize(220, 150);
```

Comment:
maybe we should also move the 150 into a variable and if where at it also the width

Reply:
done